src/components/UI/input/Input.tsx

```diff
@@ -0,0 +1,19 @@
+import classes from './Input.module.css';
+
+interface InputProps {
+    type: string;
+    min: number;
+    max: number;
+    labelText: string;
+}
+
+const Input: React.FC<InputProps> = ({type, min, max, labelText}) => {
+    return (
+        <div className={classes.inputGroup}>
+            <label>{labelText}</label>
+            <input type={type} min={min} max={max} className={classes.input}></input>
```

Comment:
also, if a component or element doesn't have `children` please close the tag without pair
```ts
<Component prop={1} prop2={''}></Component> // Bad
<input type="text"></input> // Bad
<Component prop={1} prop2={''} /> // Good
<input type="text" /> // Good
```